python/ql/src/Expressions/TruncatedDivision.ql

```diff
@@ -18,18 +18,20 @@ where
     // Only relevant for Python 2, as all later versions implement true division
     major_version() = 2
     and
-    exists(BinaryExprNode bin, Object lobj, Object robj |
+    exists(BinaryExprNode bin, Value lobj, Value robj |
```

Comment:
Now that they're values, could you rename the variables so they're not called `obj`?

Reply:
Done!